types/alcides/index.d.ts

```diff
@@ -0,0 +1,29 @@
+// Type definitions for alcides 1.4
+// Project: https://github.com/asaidimu/alcides
+// Definitions by: saidimu <https://github.com/asaidimu>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 4.6
```

Comment:
do you want to release this with support for beta version of TS only?
If you're using some specific TS features that depends on specific version, which are those?

Reply:
OK, I can see this:
https://github.com/asaidimu/alcides/blob/main/package.json#L44
but that is probably for dev, not when consuming this library

Reply:
That is an oversight on my part. Consuming the library does not require beta TS features.